app/src/main/java/com/dragic/beggining_recyclerview_kodeco/ui/AllFragment.kt

```diff
@@ -35,26 +35,38 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import com.dragic.beggining_recyclerview_kodeco.R
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.dragic.beggining_recyclerview_kodeco.databinding.FragmentAllBinding
+import com.dragic.beggining_recyclerview_kodeco.model.CreatureStore
 
 
 class AllFragment : Fragment() {
 
-  companion object {
-    fun newInstance(): AllFragment {
-      return AllFragment()
-    }
-  }
+    private lateinit var binding: FragmentAllBinding
+    private lateinit var adapter: CreatureAdapter

```

Comment:
While using a lateinit adapter is fine, we often lazily initialize it with `by lazy`. Could you refactor it that way?

Reply:
done